testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -289,160 +241,153 @@ def test_with_statement(self):
-            assert_array_almost_equal(
-                self.universe.atoms[index].velocity,
+            assert_almost_equal(
+                universe.atoms[index].velocity,
                 v_known,
                 self.prec,
-                err_msg="atom[{0:d}].velocity does not match known values".format(
-                index))
+                err_msg="atom[{0:d}].velocity does not match known values".
+                format(index))
 
 
-class _XDRNoConversion(TestCase):
-    __test__ = False
+class _XDRNoConversion(object):
     filename = None
 
-    def setUp(self):
-        self.universe = mda.Universe(PDB, self.filename, convert_units=False)
-        self.ts = self.universe.trajectory.ts
-
-    def tearDown(self):
-        del self.universe
-        del self.ts
-
+    @pytest.fixture()
+    def universe(self):
+        return mda.Universe(PDB, self.filename, convert_units=False)
 
-    def test_coordinates(self):
+    def test_coordinates(self, universe):
         # note: these are the native coordinates in nm
-        ca_nm = np.array([[6.043369675, 7.385184479, 1.381425762]],
-                         dtype=np.float32)
-        U = self.universe
+        ca_nm = np.array(
+            [[6.043369675, 7.385184479, 1.381425762]], dtype=np.float32)
+        U = universe
         T = U.trajectory
         T.rewind()
         T.next()
         T.next()
-        assert_equal(self.ts.frame, 2, "failed to step to frame 3")
+        assert_equal(universe.trajectory.ts.frame, 2,
+                     "failed to step to frame 3")
         ca = U.select_atoms('name CA and resid 122')
         # low precision match because we also look at the trr: only 3 decimals
         # in nm in xtc!
-        assert_array_almost_equal(ca.positions, ca_nm, 3,
-                                  err_msg="native coords of Ca of resid 122 "
-                                  "do not match for frame 3 with "
-                                  "convert_units=False")
+        assert_almost_equal(
+            ca.positions,
+            ca_nm,
+            3,
+            err_msg="native coords of Ca of resid 122 "
+            "do not match for frame 3 with "
+            "convert_units=False")
 
 
 class TestXTCNoConversion(_XDRNoConversion):
-    __test__ = True
     filename = XTC
 
 
 class TestTRRNoConversion(_XDRNoConversion):
-    __test__ = True
     filename = TRR
 
 
-class _GromacsWriter(TestCase):
-    __test__ = False
+class _GromacsWriter(object):
     infilename = None  # XTC or TRR
     Writers = {
         '.trr': mda.coordinates.TRR.TRRWriter,
         '.xtc': mda.coordinates.XTC.XTCWriter,
     }
 
-    def setUp(self):
-        self.universe = mda.Universe(GRO, self.infilename)
-        ext = os.path.splitext(self.infilename)[1]
-        self.tmpdir = tempdir.TempDir()
-        self.outfile = self.tmpdir.name + '/xdr-writer-test' + ext
-        self.Writer = self.Writers[ext]
+    @pytest.fixture()
```

Comment:
Could this be made class level?

Reply:
maybe. I haven't checked this extensively. It wouldn't matter here anyway. These tests don't register in out runtime. ALL the tests for issue 101 consume the most time. The new profiling options of pytest are pretty awesome to check out things like that quickly.

Reply:
I was wrong. This reduces the runtime by over  a second. This means several seconds on travis.